contracts/implementations/abstracts/RMRKAbstractEquippableImpl.sol

```diff
@@ -9,8 +9,13 @@ import "../../RMRK/utils/RMRKMintingUtils.sol";
 
 error RMRKMintZero();
 
-// Abstract implementations to reduce duplication amont implementations: RMRKEquippableImpl and RMRKEquippableImplErc20Pay
-// Having RMRKEquippableImplErc20Pay inherit and override RMRKEquippableImpl was discarded since it added a lot of size to the contract
+/**
+ * @title RMRKAbstractEquippableImpl
+ * @author RMRK team
+ * @notice Abstract implementation of RMRK equippable module.
+ * @dev This is an abstract implementation to reduce amount of implementations: `RMRKEquippableImpl` and
```

Comment:
Maybe better "extract common functionality from different implementations".  Or something like that in your words :sparkles: